### PR TITLE
For #2754 Add tab cards to share sheet

### PR DESCRIPTION
**Pull request by colintheshots**
This is using the new UX, not the old version at the top of the ticket.

device-2019-09-23-151718
device-2019-09-23-151845

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by ekager** [comment]
In the first screenshot you posted it looks like there's a white sliver of background now on the corner of the share sheet?

**Comment by sblatz** [comment]
@colintheshots Looks like this is failing on the `ShareButtonAppearanceTest` UI test.

File: app/src/main/res/layout/share_tab_item.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (boek) [inline app/src/main/res/layout/share_tab_item.xml]: nit: license

**Comment by boek** [comment]
PR has updated strings, wait until we cut the release branch to land.

**Comment by colintheshots** [comment]
@ekager There's no more white sliver now.
@sblatz The test is now fixed.
@boek This can wait to land then.
Thanks@

**Comment by colintheshots** [comment]
@ekager This needs re-review. The white next to the corners showing through has been fixed.

**Pull request merged**